docs/view_components/welcome.rst

```diff
@@ -0,0 +1,38 @@
+linking ************
+Welcome page
+************
+
+A welcome window can be seen in almost every software/website in production. The welcome window should serve as an orientation point for the user and let them know, in as brief words (and other audiovisual content) as possible, what to expect from the tool. It is also the launch point from where the user begins their energy systems simulation.
+
+The welcome page is to be implemented as an optional pop-up that springs up on the landing page when the user launches the tool. In this section, a structure for the welcome page is proposed, subject to further discussion with the stakeholders.
+
+Attributes
+----------
+
+The following are the view-components available to the user in the welcome pop-up on the landing window immediately after launching the software for the first time:
+
+#. **Welcome message on the pop-up**
+    * A text message 'Welcome to open_plan, the free and open-source energy systems planning and simulation tool based on the *OEMOF* framework and *energy cells* concept
+
+#. **Video intro message**
+    * A short, 3 minute video clip that provides the important information such as tool features, capabilities, where to find help, etc., very briefly
```

Comment:
This description of attribute goes more into the direction of a description of a view component. We are thinking about the structure of the GUI here, not its content yet. Thinking the content now is pointless because the structure might change after discussions, we need to agree to have a video first before we think about what will be on the video (I repeat myself purposefully here to make sure I make my point clear :))

Reply:
So, just mentioning the components of the welcome window are enough then right?